JavaProxy/src/main/resources/xqueries/user/find_all_reviewers.xqy

```diff
@@ -1,7 +1,7 @@
 xquery version "3.1";
 
 let $users := for $user in fn:doc("/db/test/users.xml")/users/user
-where $user/role != "Author"
+where $user/role != "Reviewer"
```

Comment:
this change should be reverted. Editor role is also Reviewer role, so reviewer roles are all except Author role